tasks/lzards-backup/README.md

```diff
@@ -0,0 +1,101 @@
+# @cumulus/lzards-backup
+
+This is a [Cumulus](https://nasa.github.io/cumulus) task which will take a list of Cumulus granule objects and based on granule collection configuration make requests to the configured LZARDS API for backup as appropriate.
+
+## Message Configuration
+
+For more information on configuring a Cumulus Message Adapter task, see
+[the Cumulus workflow input/output documentation](https://nasa.github.io/cumulus/docs/workflows/input_output).
+
+### Config
+
+There are no configuration fields defined for this task.
+
+### Input
+
+Example:
+
+The following shows the minimal set of keys required for an input payload:
+
+```json
+{
+  "granules": [
+    {
+      "granuleId": "FakeGranule001",
+      "dataType": "FakeGranuleType",
+      "version": "006",
+      "files": [
+        {
+          "filename": "s3://fakeBucket1//path/to/granule1/foo.jpg",
+          "bucket": "fakeBucket",
+          "checksumType": "md5",
+          "checksum": "someChecksum"
+        },
+        {
+          "filename": "s3://fakeBucket1//path/to/granule1/foo.dat",
+          "bucket": "fakeBucket",
+          "checksumType": "md5",
+          "checksum": "someChecksum"
+        },
+      ]
+    }
+  ]
+}
+```
+
+Each granule *must* have a `dataType` and `version` to associate it with a Cumulus collection.
+
+In addition to the task schema requirements, any granule files that are to be backed up *must* have a `checksumType` (md5 | sha256) with a value for `checksum` as LZARDS requires a checksum value.
```

Comment:
That's correct.    That's because not all granule files require them, *only* granule files that are going to be backed up by LZARDS.      Not constraining that allows a file object where some of the files have generated checksums, but not all. 

This has the downside of not resulting in an immediate CMA schema fail, before a runtime backup error.

Reply:
Yeah, I guess there's no way to enforce that at the schema level though